apps/examples/mediaplayer/WiFiConnector.cpp

```diff
@@ -0,0 +1,107 @@
+/****************************************************************************
```

Comment:
In future, we may select one of target from list.
What I meaning is that ap list of wi-fi, and bluetooth device list.
So we have to make common logic for all these connectivity.
Below is simple example
```
String *name;
ConnectivityList *list = getList(CONNECTIVITY_BLUETOOTH);
list.show();
list.setOnClickListener() {
    void onTouchUp(int id) {
        name = id.getString();
    }
 };

 Connectivity *target = getItem(list, name); //this will request to audio manager to change device ..
 UUID uuid = getUUId(target);
 BtSocket socket = createBTSocket(uuid);
 socket.connect();
```
As I wrote above, it shows what is necessary.
(getlist, making target, connect, and maybe change target, disconnect)
Maybe both of 2 connectivity will follow almost same flow, so you can make abstract class,
then you can make impl class. 
But make sure that This opinion assuming that UI. 
In this case we have to make connectivity app instead of this logic, of course app launcher is needed.
So please keep in your mind about this.

Reply:
I'll keep in mind. Thanks for the review.